tests/gateways/test_connection.py

```diff
@@ -221,9 +221,15 @@ def test_get_session_with_channel_settings_no_handler(mocker):
         "conda.gateways.connection.session.get_channel_name_from_url",
         return_value="defaults",
     )
-    mock_context = mocker.patch("conda.gateways.connection.session.context")
-    mock_context.plugin_manager.get_auth_handler.return_value = None
-    mock_context.channel_settings = ({"channel": "defaults", "auth": "dummy_two"},)
+    mock = mocker.patch(
+        "conda.plugins.manager.CondaPluginManager.get_auth_handler",
+        return_value=None,
+    )
+    mocker.patch(
+        "conda.base.context.Context.channel_settings",
+        new_callable=mocker.PropertyMock,
+        return_value=({"channel": "defaults", "auth": "dummy_two"},),
+    )
```

Comment:
With the addition of new tests above the testing order was shuffled and exposed incorrect context mocking in this unrelated test, see https://github.com/conda/conda/actions/runs/6079217031/job/16498615642#step:8:909